src/Microsoft.PowerShell.Commands.Diagnostics/GetEventCommand.cs

```diff
@@ -963,6 +963,14 @@ private string BuildStructuredQuery(EventLogSession eventLogSession)
 
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Get-WinEvent.Get-WinEvent can retrieve events.Throw if count of lognames exceeds Windows API limit
```powershell
Expected an exception, with FullyQualifiedErrorId 'LogCountLimitExceeded,Microsoft.PowerShell.Commands.GetWinEventCommand' to be thrown, but the FullyQualifiedErrorId was 'CommandNotFoundException'. from /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Diagnostics/Get-WinEvent.Tests.ps1:55 char:15
    +             { get-winevent -logname * } | Should -Throw -ErrorId "Log ...
    +               ~~~~~~~~~~~~
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Diagnostics/Get-WinEvent.Tests.ps1: line 55
55:             { get-winevent -logname * } | Should -Throw -ErrorId "LogCountLimitExceeded,Microsoft.PowerShell.Commands.GetWinEventCommand"
```